### PR TITLE
workspaces

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: developing_packages/workspaces.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    $ cd build
+    $ cmake .. -G "Visual Studio 14 Win64" # Adapt accordingly to your conan profile
+    # Now build it. You can also open your IDE and build
+    $ cmake --build . --config Release
+    $ ./A/Release/app.exe
+    Hello World C Release!
+    Hello World B Release!
+    Hello World A Release!
+
+Now the project is editable, you can change the code of C hello.cpp to say "Bye World" and:
```
Comment (danimtb) [inline developing_packages/workspaces.rst]: of folder C

File: conf.py [hunk 2]
```diff
@@ -372,3 +373,18 @@
 
 # If false, no index is generated.
 #epub_use_index = True
+
+
+redirect_files = ['creating_packages/package_dev_flow.html']
```
Comment (lasote) [inline conf.py]: Could make sense to covert it to a dict of {origin: dest} and autogenerating the redirect html?

**Comment by lasote** [comment]
I've updated develop with master, solve the conflicts.

File: conf.py (hunk 2 above)
Comment (memsharded) [reply conf.py]: Done

**Pull request merged**